docs/configuration/http_conn_man/route_config/rate_limits.rst

```diff
@@ -207,9 +208,8 @@ The configuration would be:
     ]
   }
 
-If an action doesn't append a descriptor entry, the next item in the action list will
-be processed. For example given the following rate limit configuration, a request can
-generate a few possible descriptors depending on what is present in the request.
+If an action doesn't append a descriptor entry, no descriptor is generated for the configuration.
+For example given the following rate limit configuration, there are two possible outcomes:
```

Comment:
Can you rephrase slightly. You don't actually show a second outcome. I would just move the clause after into this sentence, and show what gets output if a descriptor is output at all.

Also, this is from previous commit, but for the fixed with descriptor examples, it's a little strange to split the entries across multiple lines. If you want to line split, can you split on an entry basis and align the opening parenthesis. (Same elsewhere in this file).